ddtrace/tracer/option.go

```diff
@@ -667,8 +667,8 @@ func WithDogstatsdAddress(addr string) StartOption {
 	}
 }
 
-// WithSamplingRules specifies the sampling rates to apply to spans based on the
-// provided rules.
+// WithSamplingRules specifies the sampling rates to apply to trace spans based on the
```

Comment:
Why do you feel the `trace` word is needed here? Spans will be spans :) They all belong to traces.